common/test/acceptance/tests/lms/test_lms_course_home.py

```diff
@@ -133,3 +133,14 @@ def test_course_home_a11y(self):
         course_home_page = CourseHomePage(self.browser, self.course_id)
         course_home_page.visit()
         course_home_page.a11y_audit.check_for_accessibility_errors()
+
+    def test_course_search_a11y(self):
```

Comment:
Do we need it on the individual test as well as on the test class (line 123), or does that annotation pass down to all of the tests below?